.github/workflows/upload-pypi-source.yml

```diff
@@ -27,6 +27,7 @@ jobs:
       uses: actions/checkout@v2
       with:
         fetch-depth: 0
+        submodules: recursive
```

Comment:
This gets `mozilla-ca` into the wheel.

Reply:
I think you should specify mozilla-ca as not everyone wants the electron in the package.

Reply:
My hope is that it none of that will get dragged in because it's not explicitly listed in `setup.py`. I'll find out when the build is done.

Reply:
Looks good.